static/app/components/events/eventStatisticalDetector/eventComparisonList.tsx

```diff
@@ -0,0 +1,275 @@
+import {Fragment, useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import DateTime from 'sentry/components/dateTime';
+import {EventDataSection} from 'sentry/components/events/eventDataSection';
+import Link from 'sentry/components/links/link';
+import PerformanceDuration from 'sentry/components/performanceDuration';
+import {Tooltip} from 'sentry/components/tooltip';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Event, Group, Organization, Project} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import {Container, NumberContainer} from 'sentry/utils/discover/styles';
+import {getShortEventId} from 'sentry/utils/events';
+import {useProfileEvents} from 'sentry/utils/profiling/hooks/useProfileEvents';
+import {useProfileFunctions} from 'sentry/utils/profiling/hooks/useProfileFunctions';
+import {generateProfileFlamechartRouteWithQuery} from 'sentry/utils/profiling/routes';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface EventComparisonListProps {
+  event: Event;
+  group: Group;
+  project: Project;
+}
+
+export function EventComparisonList({event, project}: EventComparisonListProps) {
+  const evidenceData = event.occurrence?.evidenceData;
+  const fingerprint = evidenceData?.fingerprint;
+  const breakpoint = evidenceData?.breakpoint;
+  const frameName = evidenceData?.function;
+  const framePackage = evidenceData?.package || evidenceData?.module;
+
+  if (
+    !defined(fingerprint) ||
+    !defined(breakpoint) ||
+    !defined(frameName) ||
+    !defined(framePackage)
+  ) {
+    return null;
```

Comment:
I assume this should never happen right? Should we captureMessage here so we are warned in case we break this at some point?